src/main/resources/migration/triggers/function_effective-permissions-mx-on-delete.sql

```diff
@@ -10,24 +10,21 @@ Fired on DELETE FROM one of:
 CREATE OR REPLACE FUNCTION effective_permissions_mx_on_delete()
 RETURNS TRIGGER AS $$
 DECLARE
-  rec        RECORD;
-  project_id BIGINT;
+  project_ids BIGINT[];
 BEGIN
   IF TG_TABLE_NAME = 'PROJECT_ACCESS_TEAMS' THEN
-    FOR rec IN (SELECT DISTINCT "PROJECT_ID" FROM old_table) LOOP
-      PERFORM recalc_user_project_effective_permissions(rec."PROJECT_ID");
-    END LOOP;
-
+    PERFORM recalc_user_project_effective_permissions(
+      ARRAY(SELECT DISTINCT "PROJECT_ID" FROM old_table)
```

Comment:
Nit: [`ARRAY_AGG`](https://www.postgresql.org/docs/current/functions-aggregate.html#FUNCTIONS-AGGREGATE) is more idiomatic.

```suggestion
      (SELECT ARRAY_AGG(DISTINCT "PROJECT_ID") FROM old_table)
```

Note you need to wrap the `SELECT` expression in parentheses.

Reply:
This should be addressed, unless I accidentally missed any